src/style.css

```
@@ -0,0 +1,45 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1:nth-child(2) .stars__star:nth-child(-n + 1) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2:nth-child(3) .stars__star:nth-child(-n + 2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--3:nth-child(4) .stars__star:nth-child(-n + 3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--4:nth-child(5) .stars__star:nth-child(-n + 4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--5:nth-child(6) .stars__star:nth-child(-n + 5) {
+  background-image: url("images/star-active.svg");
+}
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url("images/star-active.svg");
}
```
selectors with same styles can be groped and separated by comma
selectors can be simplier